### DISCUSSION TITLE
url-http: Could not create connection to git.swf.mycompany.com:2345

### DISCUSSION THREAD

**Discussion by CDitzel** [magit/forge#466] [Support] [answered]
So I believe I set everything up correctly to use forge for my company-hosted gitlab server

1. created an access token and put the credentials in the .authinfo file
machine git.swf.mycompany.com:2345/api/v4 login cditzel^forge password ****

2. in my config file I have
(use-package forge)
(with-eval-after-load 'forge
		  (add-to-list 'forge-alist
		   '("git.swf.mycompany.com:2345"
			 "git.swf.mycompany.com:2345/api/v4"
			 "glcustom"
			 forge-gitlab-repository)))

3. Executing forge-add-repository, auto fills with ssh://git@git.swf.mycompany.com:2345/myteam.git

I get an error 

`Opening TLS connection with ‘gnutls-cli --x509cafile /etc/ssl/certs/ca-certificates.crt -p 2345 git.swf.myconpany.com --protocols ssl3’...failed
Opening TLS connection to ‘git.swf.myconpany.com’...failed
url-http: Could not create connection to git.swf.mycompany.com:2345`

any idea what this is about?

**Comment by tarsius** [comment]
2345 and 7999 are not the same number.

**Reply by CDitzel** [reply]
sorry failed to replace original port with my fake port when I adapted in for posting here. The port number is ofc fixed and consistent. So that is not the issue here

**Comment by CDitzel** [comment]
does this error speak to a more general problem like me having input the wrong credentials or is it more like sth. you have never seen before?

**Reply by tarsius** [reply]
I believe to remember that the port has to be specified separately but haven't yet gotten around to confirming that by looking at documentation and/or code.

**Comment by CDitzel** [comment]
Mh I will try to specify the port in my .ssh/config file and leave it out in the .authinfo and elisp config file

**Comment by CDitzel** [comment, marked as answer]
removing the port from both the authinfo file and emacs config file and only specifying it in the .ssh/config file worked
